classes/xt_rcar.bbclass

```diff
@@ -0,0 +1,8 @@
+rcar_unpack_proprietary_graphics() {
```

Comment:
classes/xt_rcar.inc, according to the below this is correct place for include files:
https://www.yoctoproject.org/docs/1.6/bitbake-user-manual/bitbake-user-manual.html#locating-include-and-class-files